src/components/separator/index.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from "prop-types";
+
+const COMPONENT_CLASS_NAME = "c-separator";
+
+const Separator = ({ className, separatorString }) => (
+	<span
+		className={className ? `${COMPONENT_CLASS_NAME} ${className}` : `${COMPONENT_CLASS_NAME}`}
+		// passes in the separator string to show as the content property using attr()
+		data-separator-string={separatorString}
+	/>
+);
+
+Separator.propTypes = {
+	/** Class name(s) that get appended to default class name of the component */
+	className: PropTypes.string,
+	/** Custom separator string */
+	separatorString: PropTypes.string,
+};
+
+Separator.defaultProps = {
+	separatorString: "●",
```

Comment:
With the separator output being controlled via a React Prop means any themes supplied blocks that use the default separator are not able to be changed by a client via enhanced styling. If we move the separator value to be within the Sass we lift the control to theme settings

Reply:
Allowing this prop is fine for when there is a need to have a different separator to the default